Unix/tests/cli/test_cli.cpp

```diff
@@ -537,7 +537,7 @@ static int StopServerSudo()
 
 cleanup:
     // To allow pid file to be deleted
-    Sleep_Milliseconds(200);
+    Sleep_Milliseconds(500);
```

Comment:
I've run this repeated on HP, and it seems to work.  The increase in sleep here is really to give it more time before starting next test.  Sometimes the next test starts up so fast that the "delete pid" operation deletes next test's omiserver pid file.